README.markdown

```diff
@@ -42,12 +42,12 @@ The ETM pro is released under the [MIT License](LICENSE.txt).
   cp -vn config/config.sample.yml config/config.yml
   cp -vn config/email.sample.yml config/email.yml
   ```
+  * Probably set "standalone" to `true` in "config/config.yml"
 
 * Run `bundle install` to install all the dependencies
-* Create local database with `rake db:create`
-* Fill the database with `rake db:reset`
+* Create and fill local database with `rake db:setup` and `RAILS_ENV=test rake db:setup`
```

Comment:
I know... but for me it did not work any other way - neither db:reset nor db:setup worked without the explicit environment. It is probably not worth spending time to investigate this, is it?